common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -409,6 +461,95 @@ bool mon_wlan_hal_dwpal::sta_link_measurements_11k_request(const std::string &st
     return true;
 }
 
+bool mon_wlan_hal_dwpal::channel_scan_trigger(int dwell_time_msec,
+                                              const std::vector<unsigned int> &channel_pool)
+{
+    LOG(DEBUG) << __func__ << " received on interface=" << m_radio_info.iface_name;
```

Comment:
Remove

Reply:
since its the only print for triggering scan i prefer to leave it.

Reply:
#665